### ISSUE TITLE
Plugins not working in Web Panel

### ISSUE THREAD

**Issue by darkseid-is** [zen-browser/desktop#1810]
### Captchas

- [X] I have read the instructions.
- [X] I have searched existing issues and avoided creating duplicates.
- [X] I am not filing an enhancement request.

### What happened?

Dark Reader plugin not working in Web Panel. 
Thank You.

### Reproducible?

- [X] I have checked that this issue cannot be reproduced on Mozilla Firefox.

### Version

1.0.1-a.5

### What platform are you seeing the problem on?

Linux

**Comment by salamijason** [comment]
was able to reproduce with different addons. assigned as bug

**Comment by Archer7x** [comment]
Is it not possible to treat the sidebar like a normal tab?
The Dark Reader add-on is crusial to me, especially for deepl.com in the sidebar.
Please make it work! So I can use the sidebar. I get eye-cancer from light mode.
Btw Chrome sucks. Thanks for the great Firefox browser!

**Comment by darkseid-is** [comment]
> Is it not possible to treat the sidebar like a normal tab? 

Exactly, when the sidebar is hidden and opened again the page gets fetched again. Treating the sidebar pages as normal tabs within sidebar will be quite helpful.

**Comment by Archer7x** [comment]
The ability to resize it to expand to the right would also be very helpful. I cant use whatsApp web in the sidebar because its too small and I cant see the chats. 
If the sidebar would keep the tabs open, the "sidebar apps" wouldnt need to reloat everytime and I wouldnt need to click on the last used site every singe time. :D

**Comment by darkseid-is** [comment]
Yes the issue is still relevant in latest version
